charts/trento-server/charts/trento-web/Chart.yaml

```diff
@@ -15,4 +15,4 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.0.1
+version: 1.0.2
```

Comment:
I am not sure if I am doing it right, but it seemed reasonable to update this value here since we updated `trento-web` chart